src/rgw/rgw_rados.cc

```diff
@@ -1011,42 +998,45 @@ static void obj_complete_cb(completion_t cb, void *arg)
 }
 
 
-void RGWIndexCompletionManager::create_completion(const rgw_obj& obj,
-                                                  RGWModifyOp op, string& tag,
+static std::list<cls_rgw_obj_key>
+maybe_copy_remove_objs(std::list<cls_rgw_obj_key>* const remove_objs)
```

Comment:
If possible without gigantic scope creep, I would prefer to get rid of std::lists.

(But obviously getting things working quickly is a bigger priority.)